src/OrbitQt/include/OrbitQt/orbittreeview.h

```diff
@@ -57,6 +57,16 @@ class OrbitTreeView : public QTreeView {
  public slots:  // NOLINT(readability-redundant-access-specifiers)
   void columnResized(int column, int old_size, int new_size);
 
+ protected:
+  [[nodiscard]] int sizeHintForColumn(int column) const override {
+    ORBIT_CHECK(model_ != nullptr);
```

Comment:
The Qt documentation says this function is supposed to return -1 when there is no model. I guess we should follow this guidance.

Reply:
Done.